include/asterisk/res_pjsip.h

```diff
@@ -1176,12 +1176,25 @@ void ast_sip_security_mechanisms_vector_destroy(struct ast_sip_security_mechanis
  *
  * \param security_mechanism Pointer-pointer to the security mechanism to allocate.
  * \param value The security mechanism string as defined in RFC 3329 (section 2.2)
- * \param ... in the form \<mechanism_name>;q=\<q_value>;\<mechanism_parameters>
+ *				in the form <mechanism_name>;q=<q_value>;<mechanism_parameters>
  * \retval 0 Success
  * \retval non-zero Failure
  */
 int ast_sip_str_to_security_mechanism(struct ast_sip_security_mechanism **security_mechanism, const char *value);
 
+/*!
+ * \brief Writes the security mechanisms of an endpoint into the provided buffer as a string.
```

Comment:
This doesn't actually provide a buffer, it returns a buffer.

Reply:
Done.